README.md

```diff
@@ -120,20 +155,58 @@ For example:
 
 > **Info**: Make sure you added static file handling to your pipeline by adding `app.UseStaticFiles();` to your `public void Configure` method in the `Startup.cs`, e.g. just after `app.UseHttpsRedirection();` entry. Otherwise the static file can not be found at runtime.
 
-> **Info**: Using a wwwroot sub-folder name that starts with `'.'`&nbsp;, like the suggested `.auth` folder name, is useful for content relevant only for localhost debugging as these are treated as hidden folders and are not included in publish output.
+> **Info**: Using a wwwroot sub-folder name that starts with `'.'`, like the suggested `.auth` folder name, is useful for content relevant only for localhost debugging as these are treated as hidden folders and are not included in publish output.
+
+## Auth Provider
+
+There are some predefined providers in this package. If you need your own or want contribute to our existing providers you must implement the `IEasyAuthAuthentificationService`.
+
+### `EasyAuthWithAuthMeService` (always on)
+
+This is a slightly special provider. This provider cannot be configured and does not implement the 'IEasyAuthAuthentificationService'. This provider is for development. A developer can create a JSON with the content of the `/.auth/me` endpoint of an EasyAuth Azure Web App. So you don't need a connection to the internet or azure for development and just use your local things.
```

Comment:
This isn't true. You must configure the EasyAuthWithAuthMeService for local debugging. Otherwise it won't work.